client/src/components/Lesson/LessonCreator.tsx

```diff
@@ -60,14 +58,6 @@ export class LessonCreator extends BindingComponent<LessonCreatorProps, Lesson>
         return this.state;
     }
 
-    componentWillMount() {
-        if (this.props.notFound) {
```

Comment:
It's still used on [line 80](https://github.com/teamdotjs/v2/blob/86fab4da4d7c850701980a2714e561d8ab5d1796/client/src/components/Lesson/LessonCreator.tsx#L80)